pkg/ssh/server/ssh.go

```diff
@@ -99,7 +100,20 @@ type Server struct {
 func getShell() ([]string, error) {
 	// try to get a shell
 	if runtime.GOOS != "windows" {
-		_, err := exec.LookPath("bash")
+		// infere login shell from getent
+		cmd := exec.Command("getent", "passwd", os.Getenv("USER"))
+		output, err := cmd.Output()
```

Comment:
Please create a function for this and use it like this:
```
func GetUserShell() (string, error) {
    currentUser, err := user.Current()
    if err != nil {
      return err
    }

    output, err := exec.Command("getent", "passwd", currentUser.Name).Output()
    ...
}

```

Reply:
done